scripts/test-smoke.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Script to test kube-spawn
+
+set -eux
+set -o pipefail
+
+CDIR=$(cd "$(dirname "$0")" && pwd)
+pushd "$CDIR"
+trap 'popd' EXIT
+
+if ! vagrant version > /dev/null 2>&1; then
+  echo "Please install vagrant first"
```

Comment:
Indentaions in this script are inconsistent. Sometimes you are using two spaces, sometimes tabs. Please choose the one pattern - I prefer tabs, but whatever you prefer WFM.